src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx

```diff
@@ -89,6 +95,57 @@ const TeamsTools = () => {
     </TableRow>
   ));
 
+  interface DescriptionButtonsProps {
+    toRight: ReactNode;
+  }
+
+  const DescriptionButtons: React.FC<DescriptionButtonsProps> = ({ toRight }) => {
+    return (
+      <Box
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+          marginTop: '6px',
+          marginBottom: '10px'
+        }}
+      >
+        <FormLabel>Description</FormLabel>
+        {toRight}
+      </Box>
+    );
+  };
+
+  const editButtons = (
+    <Box style={{ display: 'flex' }}>
+      <Button
+        onClick={() => {
+          setCurrentDescription(prevDescription);
+          setPrevDescription('');
+          setIsPreview(true);
+        }}
+      >
+        Cancel
```

Comment:
Currently clicking the cancel button makes the textbox disappear, which doesn't really make sense as something the user would want to do. Can you remove the cancel button altogether?